src/eko/runner.py

```diff
@@ -7,11 +7,13 @@
 
 import numpy as np
 
+from eko import compatibility
```

Comment:
Never use `eko` eplicitly inside `eko`. Use relative imports:
```suggestion
from . import compatibility
```